src/components/Missions.js

```diff
@@ -14,19 +15,53 @@ const Missions = () => {
         });
     }
   }, [dispatch, missions.length]);
+
+  const handleJoinMission = (missionId) => {
+    dispatch(joinMission(missionId));
+  };
+
+  const handleLeaveMission = (missionId) => {
+    dispatch(leaveMission(missionId));
+  };
+
   return (
-    <div>
-      <h2>Missions</h2>
-      <ul>
+    <div className="mission-container">
+      <h2 className="mission-title">Missions</h2>
+      <ul className="mission-list">
         {missions.map((mission) => (
-          <li key={mission.mission_id}>
-            <h3>{mission.mission_name}</h3>
-            <p>{mission.description}</p>
+          <li className="mission-item" key={mission.mission_id}>
+            <h3 className="mission-name">{mission.mission_name}</h3>
+            <p className="mission-description">{mission.description}</p>
+            <div className="mission-badge-container">
+              {mission.reserved ? (
+                <div className="active-member-badge">Active Member</div>
+              ) : (
+                <div className="not-a-member-badge">NOT A MEMBER</div>
+              )}
+            </div>
```

Comment:
- [ ] Good job so far❤,
- [ ]  but to follow the exact design, then I think it would be great to use HTML **table** instead of **list** or **div** tag for the Mission display section😉,